tachikoma-grpc/src/main/kotlin/com/sourceforgery/tachikoma/maildelivery/impl/MailDeliveryService.kt

```diff
@@ -237,7 +238,11 @@ private constructor(
         val subject = mergeTemplate(template.subject, globalVars, recipientVars)
         return subject to wrapAndPackBody(
             request = request,
-            htmlBody = htmlBody.emptyToNull(),
+            htmlBody = if (request.inlineCss) {
```

Comment:
Please move to wrapAndPackBody() method. It *MAY* be possible to reuse the Jsoup parsing done there